src/components/AnnotationContentOverlay/AnnotationContentOverlay.js

```diff
@@ -109,10 +101,18 @@ const AnnotationContentOverlay = () => {
     }
   };
 
-  if (isDisabled || isMobileDevice) {
+  if (isDisabled || isMobileDevice || !annotation) {
     return null;
   } else if (isUsingCustomHandler) {
-    return <CustomOverlay />;
+    return (
+      <div
```

Comment:
Do you think we should do the same for `DefaultOverlay`? I didn't actually test it but I think we also keep unmounting and then mounting them back because it's a new component every time we re-render. Since the DefaultOverlay is what most of the clients are using I think it's worth taking a further look.

Reply:
It is indeed mounting and unmounting the `DefaultOverlay`on every mouse move;  code looks not as clean but this should take care of it.